Add cleanup=False to analysis specific config

matrulda opened the pull request:
This PR adds cleanup = false for all analysis pipelines, since the setting was removed here: https://github.com/NationalGenomicsInfrastructure/miarka-provision/pull/300

Since the config no longer is related to the GRCh38 reference only, I renamed it so it is clear that it is a general analysis config.

matrulda committed:
Add cleanup=False to analysis specific config

kjellinjonas reviewed (changes requested):
Good solution. However, at the moment this will only work for RNA-seq since this is the only pipeline where we reference additional configs apart from the ones included in the aliases. The methylseq _templatea nd sarek_template files must be updated to include \_CONFIG\_ in the pipeline command.

matrulda committed:
Add general config to all templates and alternative reference to GRCh38

matrulda commented:
Great point! I just assumed that it was in all templates and just looked at rnaseq to verify :facepalm: 
I added the config to all templates and removed the parameters that specified the location of GRCh38 files, the same references/annotation files can be found under the default igenomes location.

kjellinjonas reviewed (approved):
Great, I also approved the related PR "Revert cleanup = False for upps #300" in miarka-provision now that this is fixed

matrulda commented:
Thank you!
